core/database/foxx/api/user_router.js

```diff
@@ -120,9 +240,30 @@ router
                         user_data.options = req.queryParams.options;
                     }
 
-                    var user = g_db.u.save(user_data, {
+                    user = g_db.u.save(user_data, {
                         returnNew: true,
                     });
+                    console.info(
+                        "Client:",
+                        user?._id || "unknown",
+                        "|",
+                        "Correlation_ID:",
+                        req.headers["x-correlation-id"],
+                        "|",
+                        "HTTP:",
+                        "GET",
+                        "|",
+                        "Route:",
+                        "/create",
+                        "|",
+                        "Status:",
+                        "Started",
+                        "|",
+                        "Desc:",
+                        "Creating new user entry",
```

Comment:
It would be good to include additional items in Description if they are defined. 

```
    .queryParam("name", joi.string().required(), "Name")
    .queryParam("email", joi.string().optional(), "Email")
    .queryParam("options", joi.string().optional(), "Application options (JSON string)")
    .queryParam("uuids", joi.array().items(joi.string()).required(), "Globus identities (UUIDs)")
    .queryParam("is_admin", joi.boolean().optional(), "New account is a system administrator")
```
For instance if the user is being created as an admin is an important one.